README.adoc

```diff
@@ -153,6 +153,11 @@ $ make clean
 
 Here's how to run all available tests. All tests will check all Go packages
 except those in the `vendor/` directory.
+Make sure you have docker and docker-compose available (Probably you may consider adding your user into `docker` group using `usermod -aG docker $USER`).
```

Comment:
the 'add user to docker group' does not work on Red Hat platforms due to security issues, so a -1 from my side as well to promote this.